api/Vigilance.api

```diff
@@ -392,6 +392,7 @@ public final class gg/essential/vigilance/gui/CategoryLabel : gg/essential/eleme
 	public fun <init> (Lgg/essential/vigilance/gui/SettingsGui;Lgg/essential/vigilance/data/Category;)V
 	public final fun deselect ()V
 	public final fun isSelected ()Z
+	public final fun markSelected ()V
```

Comment:
This sounds like an implementation detail (granted, the entire `CategoryLabel` does, but it's too late for that one). When would it ever make sense for a third-party to call this method (rather than `select`)? If there's no such case, then it should be `internal`, not part of our public API.